tests/bsim/bluetooth/ll/bis/testcase.yaml

```diff
@@ -0,0 +1,39 @@
+common:
+  build_only: true
+  bsim_copy_exe: true
+  tags: bluetooth
+
+tests:
+  bluetooth.ll.bis:
+    platform_allow:
+      - nrf52_bsim
+      - nrf5340bsim_nrf5340_cpunet
+    integration_platforms:
+      - nrf52_bsim
+      - nrf5340bsim_nrf5340_cpunet
+    bsim_exe_name: tests_bsim_bluetooth_ll_bis_prj_conf
+  bluetooth.ll.bis_nrf5340_cpuapp:
```

Comment:
A bit of circumstancial sidetrack: both of these can be built with sysbuild, so there is no need to differentiate them.
```diff
diff --git a/tests/bsim/bluetooth/ll/bis/testcase.yaml b/tests/bsim/bluetooth/ll/bis/testcase.yaml
index ff46037572..8f7369d5f9 100644
--- a/tests/bsim/bluetooth/ll/bis/testcase.yaml
+++ b/tests/bsim/bluetooth/ll/bis/testcase.yaml
@@ -5,18 +5,14 @@ common:
 
 tests:
   bluetooth.ll.bis:
+    sysbuild: true
     platform_allow:
       - nrf52_bsim
       - nrf5340bsim_nrf5340_cpunet
+      - nrf5340bsim_nrf5340_cpuapp
     integration_platforms:
       - nrf52_bsim
       - nrf5340bsim_nrf5340_cpunet
-    bsim_exe_name: tests_bsim_bluetooth_ll_bis_prj_conf
-  bluetooth.ll.bis_nrf5340_cpuapp:
-    sysbuild: true
-    platform_allow:
-      - nrf5340bsim_nrf5340_cpuapp
-    integration_platforms:
       - nrf5340bsim_nrf5340_cpuapp
     bsim_exe_name: tests_bsim_bluetooth_ll_bis_prj_conf
   bluetooth.ll.bis_ticker_expire_info:
```

Reply:
Changed